### ISSUE TITLE
X11 not found in macOS High Sierra

### ISSUE THREAD

**Issue by maekclena** [jcfr/VTKPythonPackage#10]
Hi,

I'm trying to build the macOS wheels but the configure step fails:
```
CMake Error at /usr/local/Cellar/cmake/3.9.6/share/cmake/Modules/FindX11.cmake:429 (message):
  Could not find X11
Call Stack (most recent call first):
  Rendering/OpenGL2/CMakeLists.txt:191 (find_package)
```

I'm under macOS High Sierra (10.13.1) where [X11 is not included](https://support.apple.com/en-us/HT201341). I believe the problem comes from setting `VTK_USE_X` to `ON` in `VTKPythonPackage/_skbuild/cmake-build/VTK-prefix/tmp/VTK-cache-Release.cmake` (line 24).

**Comment by jcfr** [comment]
Thanks for the note.

In the next few days, I will setup daily build for the VTK wheels (similar to what we have for ITK) and I will integrate your change.

**Comment by jcfr** [comment]
Closing. This should be fixed by #15
